packages/backend/server/src/mails/components/doc.tsx

```diff
@@ -10,7 +10,7 @@ export interface DocProps {
 export const Doc = (props: DocProps) => {
   return (
     <Link href={props.url}>
-      <Bold>{props.title}</Bold>
+      <Bold>{props.title || 'Untitled Doc'}</Bold>
```

Comment:
should use title `Untitled` instead of `Untitled Doc` as frontend display 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/hTwOityLamd4hitrae7M/dfabbeb0-0432-46c6-89d3-f4ed66be036e.png)